novu-style.gemspec

```diff
@@ -27,8 +27,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'rubocop', '~> 0.75.0'
-  spec.add_dependency 'rubocop-rails', '~> 2.3.2'
+  spec.add_dependency 'rubocop', '~> 0.78.0'
+  spec.add_dependency 'rubocop-rails', '~> 2.4.0'
```

Comment:
Should these be required to be stricter and require > 0.78, rather than the looser `~>`? Non blocker.

Reply:
I leave it that way because is the equivalent of having `>= 0.78.0`, `< 0.79.0`.
That being said Rubocop team usually introduces breaking changes in minor versions  and uses patches for bugfixes, this way we ensure that we won't have breaking changes for the same version of novu-style gem.